.github/workflows/ci.yml

```
@@ -0,0 +1,316 @@
+          cache: 'npm'
+
+      - name: Check for outdated dependencies
+        run: |
+          npm outdated || true
+          npx depcheck || true
+
+  # Final status check
+  ci-status:
+    name: CI Status
+    runs-on: ubuntu-latest
+    needs: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)